packages/common/src/newsletters/api/utils/authenticate-request.ts

```diff
@@ -0,0 +1,20 @@
+import { INewslettersRequestOptions } from "../types";
+
+/**
+ * return a token created using options.authentication or set on options.token
+ *
+ * @export
+ * @param {IDiscussionsRequestOptions} options
```

Comment:
`INewslettersRequestOptions`. Would you fix this in events as well? Has the same typo from discussions